tanglescope/statscollector/statscollector.cpp

```diff
@@ -115,11 +117,13 @@ void ZMQCollectorImpl::collect(uint32_t bundleConfirmationHistogramRange,
       rxcpp::observable<>::create<std::shared_ptr<iri::IRIMessage>>(
           [&](auto s) { zmqPublisher(std::move(s), _zmqURL); });
 
+  uint64_t lmsi = 0;
```

Comment:
add a comment to explain what `lmsi` stands for?

Reply:
yup